functions/getDrivers/source.js

```diff
@@ -0,0 +1,17 @@
+
+/*
+This function takes in nothing and returns the drivers associated with the organizaiton in the context.
+An example return value is [{"name":"driver1","email":"drvier1@mail.com","phone":"12341234","id":"89d22d0e-e8d1-4261-9b0a-49305788463c"}]
+*/
+
+exports = function(){
+
+  let db = context.services.get("mongodb-atlas").db("stayneighbor")
```

Comment:
In my other PR, I show how you can refer to global external config, mind updating and we can get it all merged up

Reply:
Just updated